fms/models/llama.py

```diff
@@ -243,6 +234,16 @@ def reset_params(self):
             0, 1 / math.sqrt(math.sqrt(self.width * self.shared.vocab_size))
         )
 
+        if isinstance(self.distributed_strategy, UniformModelParallelStrategy):
```

Comment:
ugh, I thought I remembered us having a fix to avoid including this code here

Reply:
I think I have a solution for this that won't require graph breaks, but it's definitely another PR by itself